engine/src/test/java/com/datatorrent/stram/StramLocalClusterTest.java

```diff
@@ -87,8 +89,19 @@ public void testLocalClusterInitShutdown() throws Exception
 
     dag.getAttributes().put(LogicalPlan.CONTAINERS_MAX_COUNT, 2);
 
+    // Add dummy jar file to classpath
+    File temp = File.createTempFile("temp", null);
+    temp.deleteOnExit();
+    dag.addJarResource(temp.getAbsolutePath(), false);
```

Comment:
This is a local mode cluster. Not the real one. The test for deleteOnCopy is added in real one.